ecosystem/sep-0008.md

```diff
@@ -249,13 +249,25 @@ If the `action_method` is provided the client uses the type of request indicated
 
    The next URL in the POST case is necessary to support products that are websites and products that are mobile applications. Products that run entirely in the browser can redirect the user to the next URL and the redirect prevents odd user experiences where a user refreshes a page and is prompted to re-post the data. Products that are mobile applications cannot open a system browser with a POST request. They can make the POST request internally, extract the redirect in the response, and open a system browser with the next URL.
 
-   Parameters:
+   Request Parameters:
+
+   Any fields defined by [SEP-9 Standard KYC / AML fields].
+
+   Request Example:
+
+   ```json
+   {
+     "email_address": "user@example.com"
```

Comment:
I've provided an example of a single field that contains no sub-fields. Technically SEP-9 contains fields like `organization.name`, and I'm not sure how they should be provided. Ideally the format of this request would be similar to SEP-12 in the sense that sub-fields are either flattened or hierarchical. I've opened https://github.com/stellar/stellar-protocol/issues/832 to find out more to understand what needs go to here, but I don't think that needs to block this PR.